src/blueapi/worker/task_worker.py

```diff
@@ -36,8 +47,11 @@
 from .worker_errors import WorkerAlreadyStartedError, WorkerBusyError
 
 LOGGER = logging.getLogger(__name__)
+TRACER = get_tracer("task_worker")
```

Comment:
Would this be a nicer convention?

```suggestion
TRACER = get_tracer(__name__)
```

Reply:
The docs explicitly tell you not to do that incase the module name is not meaningful or changes